wherehows-web/app/templates/components/application/banner-alerts.hbs

```diff
@@ -0,0 +1,15 @@
+{{#each banners.banners as |banner|}}
```

Comment:
I think `components/notifications/banner-alerts` might be more semantically "correct"? What do you think?

Reply:
Makes sense. My original thought was that `application` could be the folder that catches components related to the application route.